src/mscorlib/src/System/Guid.cs

```diff
@@ -45,18 +45,22 @@ public struct Guid : IFormattable, IComparable
         ////////////////////////////////////////////////////////////////////////////////
 
         // Creates a new guid from an array of bytes.
-        //
         public Guid(byte[] b)
+            : this(new ReadOnlySpan<byte>(b))
+        {
+        }
+
+        // Creates a new guid from a read-only span.
+        public Guid(ReadOnlySpan<byte> b)
```

Comment:
It'd be good to verify with some microbenchmarks that the performance of Guid(byte[]) hasn't suffered with this change.  This is how we want to be able to write the code, but as we're still working on span and optimizations related to it, we need to make sure we're not regressing the performance of existing code rewritten to use it.  Hopefully there's no difference and this is fine.  If there is a difference, we'll need to investigate why and what can be done about it.